Demo/Sources/SwiftUI/ToastSwiftUIDemoView.swift

```diff
@@ -0,0 +1,45 @@
+import SwiftUI
+import FinniversKit
+
+struct ToastSwiftUIDemoView: View {
+    @State var showToastFromBottom: Bool = false
+    @State var showToastFromTop: Bool = false
+
+    var body: some View {
+        VStack(spacing: .spacingM) {
+            SwiftUI.Button("Animate from top") {
+                showToastFromTop = true
+            }
+
+            ToastSwiftUIView(text: "Success", style: .success)
+            ToastSwiftUIView(text: "Action success, with a pretty long text which should wrap nicely", style: .success, action: .init(title: "Undo", action: {}))
+            ToastSwiftUIView(text: "Action success", style: .success, action: .init(title: "Action", buttonStyle: .promoted, action: {}))
+
+            ToastSwiftUIView(text: "Error", style: .error)
+            ToastSwiftUIView(text: "Action error", style: .error, action: .init(title: "Undo", action: {}))
+            ToastSwiftUIView(text: "Action error", style: .error, action: .init(title: "Undo", buttonStyle: .promoted, action: {}))
```

Comment:
Maybe the 3rd parameter could be called `actionButton` or `toastAction` instead, as `action` usually refers to a closure?